src/components/UserIcon/UserIcon.js

```diff
@@ -58,11 +58,8 @@ export default function User() {
         })}
       >
         <span className="p-contextual-menu__group">
-          <a href="#_" className="p-contextual-menu__link">
-            Profile
-          </a>
-          <a href="#_" className="p-contextual-menu__link">
-            Help
+          <a href="/docs" className="p-contextual-menu__link p-link--external">
```

Comment:
Lets use the actual destination here as you're indicating it's an external link. 
This will also prevent the link from braking when run outside of JAAS.

```suggestion
          <a href="https://jaas.ai/docs" className="p-contextual-menu__link p-link--external">
```